swig/cpp/src/Struct.h

```diff
@@ -70,19 +70,19 @@ class Val
     Val(sr_val_t *val, S_Counter counter);
     Val(const char *val, sr_type_t type = SR_STRING_T);
     Val(bool bool_val, sr_type_t type = SR_BOOL_T);
-    Val(double decimal64_val, sr_type_t type = SR_DECIMAL64_T);
```

Comment:
(I'm commenting because I'm playing with the exact same part, trying to debug SR_DECIMAL64_T in C++.)

Why is this particular ctor removed? My understanding based on the commit message is that this is some SWIG bug, is that right? One minor drawback is that it makes the C++ API a bit uglier.